doc/develop/tools/clion.rst

```diff
@@ -0,0 +1,205 @@
+.. _clion_ide:
+
+..
+   Copyright 2023 Marina Kalashina <marina.kalashina@jetbrains.com>
+
+
+CLion
+#####
+
+`CLion <https://www.jetbrains.com/clion/>`_ is a cross-platform C/C++ IDE that
+supports multi-threaded RTOS debugging.
+
+This guide describes the process of setting up, building, and debugging
+Zephyr's `threads` sample in CLion.
+
+Get CLion
+*********
+
+Download_ and install CLion.
+
+.. _Download: https://www.jetbrains.com/clion/download
+
+Initialize a new workspace
+**************************
+
+This guide gives details on how to build and debug the
+`threads` sample application, but the instructions would
+be similar for any Zephyr project and
+:ref:`workspace layout <west-workspaces>`.
+
+Before you start, make sure you have a working Zephyr development
+environment, as per the instructions in the :ref:`getting_started`.
+
+Open the project in CLion
+**************************
+
+#. In CLion, click :guilabel:`Open` on the Welcome screen or select :menuselection:`File --> Open`
+   from the main menu.
+
+#. Locate your Zephyr workspace folder (``~/zephyrproject`` if you
+   followed the Getting Started instructions), then select ``~/zephyrproject/zephyr/samples/basic/threads``
+   or another sample project folder.
+
+   Click :guilabel:`OK`.
+
+#. If prompted, click :guilabel:`Trust Project`.
+
+   See `this section`_ in CLion web help for more information on project
+   security.
+
+   .. _this section: https://www.jetbrains.com/help/clion/project-security.html#projects_security
+
+
+Configure the toolchain and CMake profile
+*****************************************
+
+#. In the **Open Project Wizard**, click :guilabel:`Add profile`.
+
+#. If the toolchain is not yet configured, click :guilabel:`Manage Toolchains` next
+   to the **Toolchain** field. We recommend that you use MinGW with default
+   settings on Windows or the System toolchain on Unix machines.
+
+#. In the toolchain settings, click :guilabel:`Add environment` and select ``~/zephyrproject/.venv/Scripts/activate.bat``.
+
+   .. image:: img/toolchain_mingw.webp
+      :width: 600px
+      :alt: MinGW toolchain with environment script
```

Comment:
please update images to figures so that there is a caption. Also, use center alignment.

```suggestion
   .. figure:: img/toolchain_mingw.webp
      :width: 600px
      :align: center
      :alt: MinGW toolchain with environment script

      MinGW toolchain with environment script
```